nix/packages/default.nix

```diff
@@ -1,15 +1,57 @@
-{
-  perSystem = { pkgs, inputs', ... }: {
-    packages.cardanow = pkgs.writeShellApplication {
-      name = "cardanow";
-      runtimeInputs = [
-        inputs'.cardano-node.packages.cardano-cli
-        inputs'.mithril.packages.mithril-client-cli
-      ];
-      text = ''
-        cardano-cli --version
-        mithril-client --version
-      '';
+{ inputs, ... }: {
+  perSystem = { pkgs, inputs', system, config, ... }:
+    {
+      packages = {
+        cleanup-local-data = pkgs.writeShellApplication {
+          name = "cleanup-local-data";
+          runtimeInputs = with pkgs; [ bash ];
+          text = ''${../../bin/cleanup-local-data.sh} "$@"'';
+        };
+        cleanup-s3-data = pkgs.writeShellApplication {
+          name = "cleanup-local-data";
+          runtimeInputs = with pkgs; [ awscli2 bash jq ];
+          text = ''${../../bin/cleanup-s3-data.sh} "$@"'';
+        };
+
+        upload-data = pkgs.writeShellApplication {
+          name = "upload-data";
+          runtimeInputs = with pkgs; [ bash awscli2 ];
+          text = ''${../../bin/upload-data.sh} "$@"'';
+        };
+
+        cardanow = pkgs.writeShellApplication
+          {
+            name = "cardanow";
+            runtimeInputs = with pkgs; [
+              inputs'.cardano-node.packages.cardano-cli
+              inputs'.mithril.packages.mithril-client-cli
+              git
+              openssh
+              config.packages.cardanow-ts
+              config.packages.upload-data
+              curl
+              jq
+            ];
+            text = ''
+              # TODO there is probably a better way to write this
+              ln -sfT ${../../nix-store-src/mithril-configurations} mithril-configurations
+              ln -sfT ${../../nix-store-src/docker-compose.yaml} docker-compose.yaml
+              ln -sfT ${inputs.cardano-configurations} cardano-configurations
```

Comment:
It's a good way IMO, the only thing I really don't like is the `nix-store-src` name.

Reply:
I'm open to suggestions here